043_bookmarks_L46.js

```
@@ -0,0 +1,67 @@
+          }
+        }
+        for (var bookmark of data) {
+          if (!bookmark.stream) {
+// {fact rule=improper-input-validation@v1.0 defects=1}
+            bookmarks.push(bookmark);
+          }
+        }
+        var results = [];
+        if (req.body.query) {
+          var regex = new RegExp(req.body.query.toLowerCase(), "g");
+          for (var bookmark of bookmarks) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'bookmark'. Consider to delete it or rename it.

'bookmark' is defined here:


(lint/suspicious/noRedeclare)